lib/ssm.ts

```diff
@@ -1,16 +1,17 @@
-import AWS from "aws-sdk";
+import { AWSError } from "aws-sdk"
+import SSM from "aws-sdk/clients/ssm";
 import { GetConfigFunc, Config } from "./types";
 
 const PAGE_SIZE = 10; // Currently the maximum page size for loading from SSM
 const MAX_PAGES = 100; // For a maximum limit of 1000 params loaded, should be plenty
 const MAX_PARAMETERS = PAGE_SIZE * MAX_PAGES;
 
-export type GetByPathParams = AWS.SSM.GetParametersByPathRequest;
-export type GetByPathResult = AWS.SSM.GetParametersByPathResult;
-export type GetByPathErr = AWS.AWSError;
+export type GetByPathParams = SSM.GetParametersByPathRequest;
+export type GetByPathResult = SSM.GetParametersByPathResult;
+export type GetByPathErr = AWSError | undefined;
 export type GetByPathCallback = (
-  err?: GetByPathErr,
-  data?: GetByPathResult
+  err: GetByPathErr,
+  data: GetByPathResult
```

Comment:
This is to fix a type error at line 67 of this file (that already existed before this PR). This also matches better the type signature from the `getParametersByPath` where those two parameters are not optional:
![image](https://user-images.githubusercontent.com/4410247/109709285-3769f800-7b94-11eb-88e0-64fceb8275c8.png)